### PR TITLE
SEOImages: User Story 1463880, Part 14

### PR DESCRIPTION
**Pull request by damabe**
## Summary

Updating image file names and references to improve SEO.

Files #1463880

Contributes to #10839

### CODE REVIEW

**Comment by damabe** [comment]
@mairaw This PR is ready for review. CATS report: https://contentqacats.azurewebsites.net/CheckResults/2019031600040/Report/

There are two image names identified by CATS as being too short. They should be handled at a later time as they are not part of the SEO image without hyphen work.

File: docs/framework/wcf/diagnostics/tracing/using-service-trace-viewer-for-viewing-correlated-traces-and-troubleshooting.md [hunk 1]
```diff
@@ -124,20 +124,18 @@ This topic describes the format of trace data, how to view it, and approaches th
...
-Graph view of WCF client activities: Ambient Activity (here 0), Process action, and Set Up Secure Session  
+ ![Graph that shows Ambient Activity and Process action.](./media/using-service-trace-viewer-for-viewing-correlated-traces-and-troubleshooting/wcf-activities-graph-ambient-process.gif)   
   
  On the upper right panel, we can see all traces related to the Process Action Add activity. Specifically, we have sent the request message ("Sent a message over a channel") and received the response ("Received a message over a channel") in the same activity. This is shown in the following graph. For clarity, the Set up Secure Session activity is collapsed in the graph.  
   
- ![Using the Trace Viewer](../../../../../docs/framework/wcf/diagnostics/tracing/media/e2etrace6.gif "e2eTrace6")  
-List of traces for the Process Action activity: we send the request and receive the response in the same activity.  
+ ![List of traces for the Process Action activity.](./media/using-service-trace-viewer-for-viewing-correlated-traces-and-troubleshooting/process-action-traces.gif)  
+ we send the request and receive the response in the same activity.
```
Comment (mairaw) [inline docs/framework/wcf/diagnostics/tracing/using-service-trace-viewer-for-viewing-correlated-traces-and-troubleshooting.md]: this sentence is now lost in the text

File: docs/framework/wcf/feature-details/how-to-create-a-workflow-service-with-messaging-activities.md [hunk 2]
```diff
@@ -16,45 +16,45 @@ This topic describes how to create a simple workflow service using messaging act
...
+     ![Screenshot that shows the default workflow in the designer.](./media/how-to-create-a-workflow-service-with-messaging-activities/default-workflow-service.jpg)  
   
      Right-click the activity labeled **Sequential Service** and select **Delete**.  
   
 ### To implement the workflow service  
   
 1.  Select the **Toolbox** tab on the left side of the screen to display the toolbox and click the pushpin to keep the window open. Expand the **Messaging** section of the toolbox to display the messaging activities and the messaging activity templates as shown in the following illustration.  
   
-     ![The toolbox with messaging tab expanded](../../../../docs/framework/wcf/feature-details/media/wfdesignertoolbox.JPG "WFDesignerToolbox")  
+     ![Screenshot that shows the toolbox with Messaging tab expanded.](./media/how-to-create-a-workflow-service-with-messaging-activities/toolbox-messaging-tab.jpg)
```
Comment (mairaw) [inline docs/framework/wcf/feature-details/how-to-create-a-workflow-service-with-messaging-activities.md]:
```suggestion
     ![Screenshot that shows the toolbox with Messaging section expanded.](./media/how-to-create-a-workflow-service-with-messaging-activities/toolbox-messaging-section.jpg)  
```

File: docs/framework/wcf/feature-details/how-to-create-a-workflow-service-with-messaging-activities.md [hunk 3]
```diff
@@ -16,45 +16,45 @@ This topic describes how to create a simple workflow service using messaging act
   
 3.  When the project is created, the Service1.xamlx file is opened in the designer as shown in the following illustration.  
   
-     ![The default workflow displayed in the designer](../../../../docs/framework/wcf/feature-details/media/defaultworkflowservice.JPG "DefaultWorkflowService")  
+     ![Screenshot that shows the default workflow in the designer.](./media/how-to-create-a-workflow-service-with-messaging-activities/default-workflow-service.jpg)
```
Comment (mairaw) [inline docs/framework/wcf/feature-details/how-to-create-a-workflow-service-with-messaging-activities.md]: I think we should specify what the screenshot is about in these alt texts

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @damabe. Changes look good. Left a few comments for you to consider.

**Comment by mairaw** [comment]
Can you look at the requested changes here @damabe?

File: docs/framework/wcf/diagnostics/tracing/using-service-trace-viewer-for-viewing-correlated-traces-and-troubleshooting.md [hunk 4]
```diff
@@ -124,20 +124,18 @@ This topic describes the format of trace data, how to view it, and approaches th
 > [!NOTE]
 >  In WCF, we show response messages being processed initially in a separate activity (Process message) before we correlate them to the corresponding Process Action activity that includes the request message, through a transfer. This happens for infrastructure messages and asynchronous requests and is due to the fact that we must inspect the message, read the activityId header, and identify the existing Process Action activity with that id to correlate to it. For synchronous requests, we are blocking for the response and hence know which Process action the response relates to.  
   
- ![Using the Trace Viewer](../../../../../docs/framework/wcf/diagnostics/tracing/media/e2etrace4.gif "e2eTrace4")  
-WCF client activities listed by creation time (left panel) and their nested activities and traces (upper right panel)
```
[the remark this answers is not in the document]
Comment (damabe) [reply docs/framework/wcf/diagnostics/tracing/using-service-trace-viewer-for-viewing-correlated-traces-and-troubleshooting.md]: Made this change. Will apply to other images here (and other PRs, as needed). On the alt text, if I add the text as you recommend, it ends up higher than the 70 character threshold. I modified to be <70. Please review and let me know if it is OK.

File: docs/framework/wcf/feature-details/how-to-create-a-workflow-service-with-messaging-activities.md (hunk 3 above)
Comment (damabe) [reply docs/framework/wcf/feature-details/how-to-create-a-workflow-service-with-messaging-activities.md]: @mairaw I've updated this one and the next (Messaging tab > Messaging section) based on your feedback. I'm not sure if I should make additional changes to the alt text content in the rest of this article. It seems that's what you're asking me to do. I'm just not sure what to change them to. Please provide guidance and I'll make the changes.

**Comment by damabe** [comment]
@mairaw Made the changes. Please review and provide feedback. Updated CATS: https://contentqacats.azurewebsites.net/CheckResults/2019032300028/Report/

**Review by mairaw — APPROVED** [review]
Thx @damabe!

**Pull request merged**